app/models/petition.rb

```diff
@@ -343,7 +343,7 @@ def mark_petitions_as_debated!(date = Date.current)
     end
 
     def archived?
```

Comment:
Hmm, I guess to follow Rails naming convention it should be `archived_all?` similar to `update` vs. `update_all` and `destroy` vs. `destroy_all`.

I think maybe just having an `unarchived` scope and moving the check to the `Parliament` model would be clearer.